.gitignore

```diff
@@ -45,4 +45,5 @@ wp-content/themes/pawar2018/style.css.map
 wp-content/themes/pawar2018/js/main.min.js.map
 wp-content/themes/pawar2018/js/main.min.js
 
-wp-content/themes/pawar2018/js/main.min.js.map
\ No newline at end of file
+wp-content/themes/pawar2018/js/main.min.js.map
+wp-content/themes/pawar2018/assets/
```

Comment:
Is your editor not configured to add a new line at the end of the file? Not big deal, but that "No newline..." warning is going to come up whenever you create a new file or modify one without a newline at the end of the file already.